### ISSUE TITLE
An ATSC 3.0 receiver manufacturer reports the WMAS:2019 test landing page generates an error when launched via a PCAP containing an ATSC 3.0 HELD pointing at "https://webapitests2019.ctawave.org/wave/"

### ISSUE THREAD

**Issue by bobcampbell-eurofins** [cta-wave/WMAS#66]
> {"error": {"message": "u'/'\nTraceback (most recent call last):\n  File \"/home/ubuntu/WMAS/tools/wptserve/wptserve/server.py\", line 266, in finish_handling\n    handler(request, response)\n  File \"/home/ubuntu/WMAS/tools/serve/serve.py\", line 396, in __call__\n    wave_server.handle_request(request, response)\n  File \"/home/ubuntu/WMAS/tools/wave/network/http_handler.py\", line 48, in handle_request\n    self.handle_static_file(request, response)\n  File \"/home/ubuntu/WMAS/tools/wave/network/http_handler.py\", line 74, in handle_static_file\n    self.static_handler.handle_request(request, response)\n  File \"/home/ubuntu/WMAS/tools/wave/network/static_handler.py\", line 41, in handle_request\n    (u\"Content-Type\", content_types[file_path.split(u\".\")[-1]]))\nKeyError: u'/'\n", "code": 500}}

This was launched via our PCAP which points the ATSC 3.0 user agent at  https://webapitests2019.ctawave.org/wave/

This seemed to be a similar trace as we saw on another ATSC 3.0 implementation, from another manufacturer. We weren't sure where the problem lay, as haven't been able to follow up with them yet (as in: they've not answered our questions).

[Also they asked should the URL not be:
 **https://webapitests2019.ctawave.org/wave** ? Not sure what practical difference it would make, but for usability the WMAS website should probably automatically redirect from no-slash to slash variant?]

**Comment by JohnRiv** [comment]
WMAS2019 is still in development, so that URL is still the updated 2018 tests. Once WMAS2019's test suite is deployed, this can be retested.

**Comment by JohnRiv** [comment]
Also, good point about https://webapitests2019.ctawave.org/wave. We should have that redirect to https://webapitests2019.ctawave.org/wave/ or fix the paths to the resources.

**Comment by louaybassbouss** [comment]
> Also, good point about https://webapitests2019.ctawave.org/wave. We should have that redirect to https://webapitests2019.ctawave.org/wave/ or fix the paths to the resources.

@JohnRiv  @bobcampbell-eurofins this is fixed in WMAS2020 and later also in WMAS2019. Once we deloy the WMAS2019 Test Runner to AWS, the problem should be solved.

**Comment by JohnRiv** [comment]
Update: the WMAS2019 test suite is deployed and available, and the /wave redirect is now in place. However, it would be best to wait until @louaybassbouss and his team perform their testing and confirm it is working as expected before rerunning the ATSC test

**Comment by bobcampbell-eurofins** [comment]
Any update on this?

**Comment by JohnRiv** [comment]
@bobcampbell-eurofins the test suite is ready for verification

**Comment by bobcampbell-eurofins** [comment]
We've checked and the basic "run all the tests" seems to work now via the PCAP we created which launches the 2019 runner/tests in an ATSC 3.0 user agent/application environment. But as discussed at length in the ad-hoc joint call, devices can't run/pass all the tests as some aren't viable on these platforms. 

We'll look out for the subset that we all agreed was necessary and can validate that when its available? Let us know if we can help otherwise. I'll mark this as closed and track the ongoing activity in #63
